templates/patterns/field_list/pattern-field-list--variant-horizontal.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
+/**
+ * @file
+ * Field list horizontal variant.
+ */
+#}
+
+{% include '@ecl/description-list' with {
+  'variant': 'horizontal',
```

Comment:
We don't need to define a different template for the variant, the variant is a variable that is available so we can simply pass it on the default template.